Snippets/Operations/SqlServer/NativeSendTests.cs

```diff
@@ -49,6 +51,48 @@ public void Send()
             }
         }
 
+        [Test]
+        public void SendPowershell()
+        {
+            State state = new State();
+            using (IBus bus = StartBus(state))
+            {
+                string message = @"{ Property: 'Value' }";
+
+                var headers = new Dictionary<string, string>
+                {
+                    {"NServiceBus.EnclosedMessageTypes", "Operations.SqlServer.NativeSendTests+MessageToSend"}
+                };
+
+                // Run a PowerShell script with two parameter and set execution policy to Unrestricted
+                RunPowershellScript(@"C:\Code\docs.particular.net\Snippets\Operations\SqlServer\NativeSendPowershell.ps1", connectionString, endpointName, message, headers);
+
+                state.ResetEvent.WaitOne();
+            }
+        }
+
+        private static void RunPowershellScript(string scriptFile, params object[] parameters)
+        {
+
+            RunspaceConfiguration runspaceConfiguration = RunspaceConfiguration.Create();
+
+            using (Runspace runspace = RunspaceFactory.CreateRunspace(runspaceConfiguration))
+            {
+                runspace.Open();
+                RunspaceInvoke scriptInvoker = new RunspaceInvoke(runspace);
+                scriptInvoker.Invoke("Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force");
```

Comment:
@petersgiles This changes the powershell security permanantly for anyone who runs the unit test to allow any PS1  to run from anywhere.  I'd suggest changing the scope to "Process" rather than current user